python_modules/dagster/dagster/_core/blueprints/base_asset_blueprint.py

```diff
@@ -0,0 +1,43 @@
+from abc import abstractmethod
+from typing import AbstractSet, Any, Mapping, Optional, Sequence
+
+from dagster import multi_asset
+from dagster._core.blueprints.blueprint import Blueprint, BlueprintDefinitions
+from dagster._core.execution.context.compute import AssetExecutionContext
+from dagster._core.models import DagsterModel
+
+
+class AssetSpecModel(DagsterModel):
+    key: str
+    deps: Sequence[str] = []
+    description: Optional[str] = None
+    metadata: Mapping[str, Any] = {}
+    group_name: Optional[str] = None
+    skippable: bool = False
+    code_version: Optional[str] = None
+    owners: Sequence[str] = []
+    tags: Mapping[str, str] = {}
+
+
+class BaseMultiAssetBlueprint(Blueprint):
```

Comment:
My feeling is that this isn't the right place to innovate on terminology - for now I think we should actually probably just call this `AssetsDefinitionBlueprint`. If we settle on a different term in core Dagster, it will be easy to change this.

Reply:
If you have time tomorrow, let's find some time to talk through it? Probably easier to resolve in a higher bandwidth communication medium.

Reply:
Given that we have `BlueprintDefinitions` how about we just do `BlueprintAssetsDefinition` and then repeat the `Blueprint` prefix if we do any more of this specializations (e.g. `BlueprintAssetChecksDefinition`) we have an easy pattern. Then we can keep blueprints in in sync as we evolve the core ontology?